samples/bluetooth/peripheral_power_profiling/src/main.c

```diff
@@ -628,10 +626,16 @@ static void system_off(void)
 
 	dk_set_led_off(RUN_STATUS_LED);
 
-	int rc = pm_device_action_run(cons, PM_DEVICE_ACTION_SUSPEND);
+	if (IS_ENABLED(CONFIG_PM_DEVICE)) {
```

Comment:
Can you add `&& IS_ENABLED(CONFIG_SERIAL)` to that `if`? It will allow to disable UART with single change in Kconfig (CONFIG_SERIAL=n).

Reply:
Good point. Thanks.